src/sgame/sg_spawn.cpp

```diff
@@ -1024,15 +1024,17 @@ static void InitDisabledItemCvars()
 	G_SpawnStringIntoCVar( "disabledBuildables", g_disabledBuildables );
 }
 
+static std::string defaultTacticBehaviors = "default, defend, attack, stay_here, follow";
 static void InitTacticBehaviorsCvar()
 {
 	static Cvar::Callback<Cvar::Cvar<std::string>> g_tacticBehaviors(
 	   "g_tacticBehaviors",
 		"Allowed behaviors for the tactic command, example: " QQ("default, camper, reckless"),
 		Cvar::NONE,
-		"",
+		defaultTacticBehaviors.c_str(),
 		BG_SetTacticBehaviors
 		);
+	BG_SetTacticBehaviors( defaultTacticBehaviors.c_str() );
```

Comment:
Why force it back to the default value at the beginning of every match? Seems wrong.

Reply:
Hmm, is it really? My first try was to simply give the default value as 4th argument, like in

```
	static Cvar::Callback<Cvar::Cvar<std::string>> g_tacticBehaviors(
		"g_tacticBehaviors",
		"Allowed behaviors for the tactic command, example: " QQ("default, camper, reckless"),
		Cvar::NONE,
		"default, defend, attack, stay_here, follow",
		BG_SetTacticBehaviors
		);
```

But it turns out that the callback function is not called automatically when doing this. The result is an inconsistency between the cvar's value string and the data structure holding the values for actual use.

Reply:
Nevermind. I made sure this can only be done once.